packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -107,7 +122,7 @@ export function handler(app: Application) {
       }
       logger.info(`Determined ${repo} is ${language}`);
 
-      config = languageConfig[language];
+      config = extend(true, {}, languageConfig)[language];
```

Comment:
nit, you could probably move the indexed value into extend so we don't have to copy extra entries.

Reply:
I'll suffer the inefficiency